### ISSUE TITLE
PF upgrade |  error in SQL syntax near '/ DELIMITER'  | DB upgrade script from 6.0.0 to 6.1.0

### ISSUE THREAD

**Issue by piellick** [inverse-inc/packetfence#1573]
hello,

i follow this procedure (based on your guide https://github.com/inverse-inc/packetfence/blob/stable/UPGRADE.asciidoc ) to upgrade my pf install from 6.0.3 to 6.1.0 on debian 8u5.

mysqd from packetfence package : 

`mysqld (mysqld 5.5.49-0+deb8u1)`

```
service packetfence stop
service packetfence-config stop
```

```
apt-get ugrade 
apt-get install packetfence
```

`mysql -u root -p pf -v < /usr/local/pf/db/upgrade-6.0.0-6.1.0.sql`

i had error, on Mysql schéma upgrade script near DELIMITER entry

You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '/
DELIMITER' at line 103 SQL.sql 199 1 

For executing SQL script, i had to modify the three  delimiter ending command from

```
END /
DELIMITER ;
```

to

```
END 
/
DELIMITER ;
```

After that script was executed correctly, i did : 

modify`/usr/local/pf/conf/currently-at`to value 

`PacketFence 6.1.0`

I run the following command : 

```
/usr/local/pf/bin/pfcmd pfconfig clear_backend
service packetfence-config restart
service packetfence restart
```

but i had still the error : 

```
juin 22 12:22:38 packetfence pfcmd[3010]: Checking configuration sanity...
juin 22 12:22:44 packetfence pfcmd[3010]: FATAL - The PacketFence database schema version '6.0.0' does not match the current installed version '6.1.0'
juin 22 12:22:44 packetfence pfcmd[3010]: Please refer to the UPGRADE guide on how to complete an upgrade of PacketFence
juin 22 12:22:44 packetfence systemd[1]: packetfence.service: control process exited, code=exited status=255
juin 22 12:22:45 packetfence systemd[1]: Failed to start PacketFence Service.
juin 22 12:22:45 packetfence systemd[1]: Unit packetfence.service entered failed state.
```

How can i check if my db schema is up to date ?

**Comment by louismunro** [comment]
Try running these statements in the database:

```
SET @MAJOR_VERSION = 6;
SET @MINOR_VERSION = 1;
SET @SUBMINOR_VERSION = 0;

--
-- The VERSION_INT to ensure proper ordering of the version in queries
--

SET @VERSION_INT = @MAJOR_VERSION << 16 | @MINOR_VERSION << 8 | @SUBMINOR_VERSION;
--
-- Updating to current version
--

INSERT INTO pf_version (id, version) VALUES (@VERSION_INT, CONCAT_WS('.', @MAJOR_VERSION, @MINOR_VERSION, @SUBMINOR_VERSION));
```

I can't replicate the issue with the delimiter.
Both ways work on my install.

**Comment by piellick** [comment]
It works thanks luis. If you need some debug trace ...ask me.

I have now the good entry version on pf_version table.

**Comment by louismunro** [comment]
Fixed in 6.1.1.
